tabs/tab_corr.py

```
@@ -1,23 +1,26 @@
+- Detailed interpretations (Fixed ICC display issue)
 
 Updated: Uses dataset selector pattern like tab_diag.py
 """
 
 from shiny import ui, reactive, render, req, module
+from shinywidgets import output_widget, render_widget  # ✅ Import shinywidgets
 import pandas as pd
 import numpy as np
 import correlation  # Import from root
 import diag_test  # Import for ICC calculation
 from typing import Optional, List, Dict, Any, Union, cast
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)